businesscentral-monitoring/tests/features/businesscentral-monitoring.feature

```diff
@@ -1,4 +1,4 @@
-@rhpam-7/rhpam78-businesscentral-monitoring
+@rhpam-7/rhpam7.9-businesscentral-monitoring
```

Comment:
There shouldn't be '.', right?
```suggestion
@rhpam-7/rhpam79-businesscentral-monitoring
```

Reply:
not really, there should not be 7.9 or 79 in the image name, these tests shouldn't be here anyways. @errantepiphany do you remember why we kept these tests here?

Reply:
updated to the correct images name.
after we merge this will move these tests to jboss-kie-modules.
@errantepiphany do you agree with it?